packages/examples/src/react-next/__snapshots__/Keytips.CommandBar.Example.tsx.shot

```diff
@@ -1,674 +1,855 @@
-                        display: inline-block;
-                      }
-                      {
-                        -moz-osx-font-smoothing: grayscale;
-                        -webkit-font-smoothing: antialiased;
-                        font-family: "FabricMDL2Icons";
-                        font-style: normal;
-                        font-weight: normal;
-                        speak: none;
-                      }
-                      {
-                        color: #605e5c;
-                        flex-shrink: 0;
-                        font-size: 12px;
-                        height: 16px;
-                        line-height: 16px;
-                        margin-bottom: 0;
-                        margin-left: 4px;
-                        margin-right: 4px;
-                        margin-top: 0;
-                        text-align: center;
-                      }
-                      @media screen and (-ms-high-contrast: active){& {
-                        color: GrayText;
+                  <i
+                    aria-hidden={true}
+                    className=
+                        ms-Icon
+                        ms-Button-menuIcon
+                        {
+                          display: inline-block;
+                        }
+                        {
+                          -moz-osx-font-smoothing: grayscale;
+                          -webkit-font-smoothing: antialiased;
+                          font-family: "FabricMDL2Icons";
+                          font-style: normal;
+                          font-weight: normal;
+                          speak: none;
+                        }
+                        {
+                          color: #605e5c;
+                          flex-shrink: 0;
+                          font-size: 12px;
+                          height: 16px;
+                          line-height: 16px;
+                          margin-bottom: 0;
+                          margin-left: 4px;
+                          margin-right: 4px;
+                          margin-top: 0;
+                          text-align: center;
+                        }
+                        @media screen and (-ms-high-contrast: active){& {
+                          color: GrayText;
+                        }
+                    data-icon-name="ChevronDown"
+                    role="presentation"
+                    style={
+                      Object {
+                        "fontFamily": "\\"FabricMDL2Icons\\"",
                       }
-                  data-icon-name="ChevronDown"
-                  role="presentation"
-                  style={
-                    Object {
-                      "fontFamily": "\\"FabricMDL2Icons\\"",
                     }
+                  >
+                    
+                  </i>
+                </span>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>,
+  <span
```

Comment:
Did this PR somehow accidentally change Modal so it's by default? That would be my guess from looking at this snapshot.

Reply:
Thanks for mentioning this! There turned out to be a bug in `Modal's` dismissal.